projects/br_scs/src/Mirza/BusinessRegistry/Main.hs

```diff
@@ -151,13 +154,16 @@ initBRContext opts@(ServerOptionsBR dbConnStr _ _ _ lev mlogPath envT) = do
                       60 -- How long in seconds to keep a connection open for reuse
                       10 -- Max number of connections to have open at any one time
                       -- TODO: Make this a config paramete
-  pure $ BRContext envT connpool params logEnv mempty mempty
+  eitherJwk <- eitherDecodeFileStrict "auth_public_key_2019-04-01.json"
+  let makeError errorMessage = error $ "Unable to get the OAuth Public Key. Error was: " <> (show errorMessage)
+  let jwk = either makeError id eitherJwk
```

Comment:
`either _ id` = `fromRight`
`either id _` = `fromLeft`

Reply:
`fromRight :: b -> Either a b -> b ` just takes the `b`, but here I need the signature to pass the `a` to turn to a `b` since `makeError` requires the `eitherJwk`.